### DISCUSSION TITLE
Paperless-NGX Database import failed

### DISCUSSION THREAD

**Discussion by moe215463** [tteck/Proxmox#3266] [Q&A]
Hello 

I used the Proxmox helper script for Paperless-NGX

I have the following problem, I want to move my Paperless-NGX to a new server. I made an export on the old server, as in the instructions. Installed a new LXC container with the Proxmox helper script. Now I get the following error /opt/paperless/src# python3 manage.py document_importer /root/backup_file/

Found existing user(s), this might indicate a non-empty installation
Checking the manifest
Database import failed
No version information present

How can I install a new LXC container with Proxmox helper script for Paperless-NGX without user admin?
Or how else can I best solve this?

Paperless-ngx-Version
    2.10.2
Installationstyp
    bare-metal

Datenbank
Typ
    postgresql
Status
    OK

Migrationsstatus
    Aktuell

Thanks to all!

**Comment by digitaljdr** [comment]
are you trying to import a .zip? they don't support zips 

https://docs.paperless-ngx.com/administration/#importer

**Comment by moe215463** [comment]
Hello
No, I didn't import it as a ZIP file. I imported this following the instructions.
